lib/node_modules/@stdlib/utils/error-to-json/README.md

```diff
@@ -0,0 +1,243 @@
+# Error to JSON
+
+> Return a [JSON][json] representation of an [error][js-error] object.
+
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var toJSON = require( '@stdlib/utils/error-to-json' );
```

Comment:
Options:

```
toJSON
errorToJSON
toJSONError
err2json
error2json
errToJSON
```
Ideally, we would have a name which more or less corresponds to what will be exported to namespace.